api/course/courseModel.js

```diff
@@ -0,0 +1,29 @@
+const db = require('../../data/db-config');
+
```

Comment:
Nice names for the DB functions. This will definitely help the other teams in the future.